packages/thunderstore-api/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * TODO: this is a naive implementation. Actual process should include:
+ * - Typesafe parsing of URLs from parameters
+ * - Using env variable to define backend
```

Comment:
I'd prefer the function itself here does not depend on any globals, but rather takes the necessary configs as parameters somehow (probably as an object with multiple potential keys to make partial re-use easier for callers.)

Reply:
Updated the comment so I'll remember this.